cogs5e/sheets/dicecloudv2.py

```diff
@@ -706,9 +706,7 @@ def parse_attack(self, atk_prop) -> Attack:
         verb, proper = (
             ("casts", True)
             if atk_prop["type"] == "spell"
-            else ("uses", False)
-            if atk_prop["actionType"] != "attack"
-            else (None, False)
+            else ("uses", False) if atk_prop["actionType"] != "attack" else (None, False)
```

Comment:
this change _looks_ pretty bizarre to me, but it might not affect anything

Reply:
I'm sadly, not finding any documentation on a configuration flag we can use to disable this particular check with Black.

Reply:
i guess we can either leave it and let black do black to it, or we can try to rewrite it as a more pythonic if/else block?

i think this would do the same thing:
```
verb, proper = None, False
if atk_prop["type"] == "spell":
    verb, proper = "casts", True
elif atk_prop["actionType"] != "attack":
    verb, proper = "uses", False
```

it's not important anyway, just a formatting preference where black and i differ xD